Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -0,0 +1,264 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+import requests
+import tempfile
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+ALERT_TITLE = 'Prisma Cloud Compute Alert - '
+ALERT_TYPE_VULNERABILITY = 'vulnerability'
+ALERT_TYPE_COMPLIANCE = 'compliance'
+ALERT_TYPE_AUDIT = 'audit'
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+class Client(BaseClient):
+    """Client to use in integrations with powerful _http_request
+            :type base_url: ``str``
+            :param base_url: Base server address with suffix, for example: https://example.com/api/v2/.
+
+            :type verify: ``str``
+            :param verify: Either a boolean, in which case it controls whether we verify
+            the server's TLS certificate, or a string, in which case it must be a path
+            to a CA bundle to use..
+
+            :type proxy: ``bool``
+            :param proxy: Whether to run the integration using the system proxy.
+
+            :type ok_codes: ``tuple``
+            :param ok_codes:
+                The request codes to accept as OK, for example: (200, 201, 204).
+                If you specify "None", will use requests.Response.ok
+
+            :type headers: ``dict``
+            :param headers:
+                The request headers, for example: {'Accept`: `application/json`}.
+                Can be None.
+
+            :type auth: ``dict`` or ``tuple``
+            :param auth:
+                The request authorization, for example: (username, password).
+                Can be None.
+
+            :return: No data returned
+            :rtype: ``None``
+            """
+
+    def __init__(self, base_url, verify, proxy=False, ok_codes=tuple(), headers=None, auth=None):
+        if verify in ['True', 'False']:
+            super().__init__(base_url, str_to_bool(verify), proxy, ok_codes, headers, auth)
+        else:
+            # verify points a path to certificate
+            super().__init__(base_url, True, proxy, ok_codes, headers, auth)
+            self._verify = verify
+
+    def test(self):
+        """
+        Sends a test request to check connectivity, authentication and authorization
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix='',
+            params={'to': time.strftime('%Y-%m-%d', time.gmtime(0))})
+
+    def list_incidents(self):
```

Comment:
I think this function should have `to` and `from` arguments.
with defaults `None`.
if they are not provided then they are not passed. The test can reuse `self.list_incidents(to=time.strftime('%Y-%m-%d', time.gmtime(0)))`

Reply:
Good idea, tnx! Done.